torchmdnet/models/torchmd_et.py

```diff
@@ -79,7 +79,12 @@ class TorchMD_ET(nn.Module):
             (default: :obj:`False`)
         check_errors (bool, optional): Whether to check for errors in the distance module.
             (default: :obj:`True`)
-
+        additional_labels (Dict[str, Any], optional): Define the additional method to be used by the model, and the parameters to initialize it.
+            example:
+            additional_labels = {method_name1: {label_name: 'extra_arg_label', 'method_prm1': method_prm1,  'method_prm2': method_prm2}, 
+                                 method_name2: {label_name: 'extra_arg_label', 'method_prm1': method_prm1,  'method_prm2': method_prm2},
+                                 ...}
+            (default: :obj:`None`)
```

Comment:
I believe there should be no mention to additional_labels whatsoever in ET. Unless you go ahead and implement an actual method (like peters embedding for instance). As it is just now it is just misleading, no matter what I send in this argument it is just going to be silently ignored.
I believe you should at least implement a check of the method names sent to the representation models. For instance, if "TensornetQ" is received by ET that should be an error.
I understand you might want to add this argument for opening the door to future ways of handling the additional labels, but at least you should do something like:
```python
assert additional_labels is None, "ET does not support this feature"
```
The docstring should also reflect this current state of things.

Reply:
I will add that assertion to all the models which are not using additional_labels for now. The check on the method name it's done by the initialize_additional_method function (like the tensornet one). I can add it for each model if you want but at the moment since any additional_method is available (except for tensornet) I would say the assertion it's fine